tests/baselines/reference/plainJSGrammarErrors.types

```diff
@@ -606,7 +606,7 @@ export let noMeta = import.metal
 function foo() { new.targe }
 >foo : () => void
 >new.targe : () => void
->targe : any
+>targe : () => void

```

Comment:
Weird, how is this changing? This doesn't seem to be using meta at all.

Reply:
`new.something` is also a `MetaProperty` node, I'm guessing this is to also support `new.target`.

Reply:
Probably fall-out logic where any meta-property that's not `import.meta` is assumed to be `new.target`.